pymongo/__init__.py

```diff
@@ -155,3 +155,19 @@ def timeout(seconds: Optional[float]) -> ContextManager:
     if seconds is not None:
         seconds = float(seconds)
     return _csot._TimeoutContext(seconds)
+
+
+import os
+
+from bson.objectid import ObjectId
```

Comment:
We should `import ObjectId as _ObjectId` so it isn't implicitly in the `pymongo` namespace.